Dockerfile

```diff
@@ -6,8 +6,32 @@ RUN sudo pacman --noconfirm --needed -S \
   chromium \
   firefox \
   xorg-server-xvfb \
-#  x11vnc \
+  x11vnc \
   nodejs \
+  ### chrome dependencies
+  alsa-lib \
+  desktop-file-utils \
+  flac \
+  gconf \
+  gnome-themes-standard \
+  gtk2 \
+  harfbuzz \
+  harfbuzz-icu \
+  hicolor-icon-theme \
+  libpng \
+  libxss \
+  libxtst \
+  nss \
+  opus \
+  snappy \
+  speech-dispatcher \
+  ttf-font \
+  xdg-utils \
+  fakeroot \
+  patch \
+  # yaourt
   openssh
 
+RUN git clone https://aur.archlinux.org/google-chrome.git && cd google-chrome && makepkg && sudo pacman -U google-chrome-5* --noconfirm --needed
```

Comment:
Can you just use Chromium? Does that install the correct dependencies? Why do you need `google-chrome`?